src/app/component/event-feed/event-feed.component.ts

```diff
@@ -22,13 +24,12 @@ export class EventFeedComponent {
   @Output()
   followChanged: EventEmitter<string> = new EventEmitter<string>();
 
-  private ndkProvider: NdkproviderService;
-  private topicService: TopicService;
+  users:User[] = [];
   followedTopics: string[]|undefined;
   events: Set<NDKEvent> | undefined;
   nextEvents: Set<NDKEvent> | undefined;
   ngOnInit() {
-    this.ndkProvider.followedTopicsEmitter.subscribe((followedTopics: string) => {      
+    this.ndkProvider.followedTopicsEmitter.subscribe((followedTopics: string) => {
```

Comment:
Not sure how this will compile when the entir ndkProvider variable is deleted above :)  there will be no this.ndkProvider

Reply:
Whenever we specify public or private on a constructor parameter a corresponding public/private variable is created on the class and filled with the value of the parameter. So this will be done automatically and we will be able to reference the variable with this and the variablename given in the constructor